### PR TITLE
ports white kit changes from tg

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

most originalest pr: https://github.com/tgstation/tgstation/pull/47092

other changes since to make everything look nicer and adding some of the related items but it gives the main idea

tldr medical healing items heal slightly slower and for less but have more uses, upgraded variants available with cooperation from botany

can't test this locally because monstermos has decided to not work on local because :^^^^^^^^

new chemical cellulose made from grinding wood and wood related stuff like paper
new plant aloe does aloe stuff can be juiced for aloe juice
replaces bruise packs and ointment with sutures and regenerative mesh, which heal 10 damage per use down from 40 but have 15 uses up from 6
upgraded variants can be synthesized with some chemicals from certain botany plants that heal 15 points per use synthesized with
cellulose 10, formaldehyde 20, polypyr 15 = 1 stack of medicated suture
cellulose 20, aloe juice 20, sterilizine 10 = 1 stack of advanced regenerative mesh

access to certain plants will allow weaker variants, aloe cream, which heals 3 points per use, has 20 uses and is made from cooking aloe with a microwave or bonfire grille,

and mourning poultice which is made from an assortment of plant chemicals:
amanitin 10, cellulose 20, aloe juice 20 = 1 mourning poultice note the TG variant uses bungo toxin but bungo toxin causes heart damage and I don't like that especially because its metabolism is SLOW so we don't have it instead using amanitin which is similarly lethal but has some actual game to its game
or
entpoly 15, cellulose 20, tinlux 12 = 1 mourning poultice so fardwalkers can use it
which heals 10 brute AND burn but causes oxygen damage equal to the amount healed and can be used on corpses

### Why is this change good for the game?
makes medbay slightly less reliant on chem for over time healing and pushes cheap healing away from being a surgery only thing
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
stack healing items have been changed to heal less per use but have more uses, newly available upgraded or alternate variants are craftable usually with botany stuff so I can make department purists cry, recipes are above
also using these meds on others CURRENTLY now has a small time penalty rather than being instant

### What should players be aware of when it comes to the changes your PR is implementing?
healing takes slightly longer with white kit stuff but you don't have to worry about wasting one of your 12 bruise packs on anything less grievous than a wound from a .50 cal
### What general grouping does this PR fall under? 
medical and botany

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
baseline healing items now heal 10 points and have 15 uses, rather than 40 points and 6 per stack
upgraded variants are available with recipes previously mentioned that heal 15 points per use
aloe cream available from microwaving aloe which heals 3 points but has 2 uses per aloe microwaved
mourning poultice craftable with lavaland mushrooms or some debatably dangerous botany chemicals that heal 10 brute AND burn but deal oxygen damage based on their healing

# Changelog

:cl:  lots of tg coders
rscadd: several new weaker but more numerous healing stack items, including recipes for unique and upgraded variants
rscdel: old healing stack items are now unattainable since they are boring
imageadd: icons for all the new stuff
soundadd: sounds for poultice
/:cl:

### CODE REVIEW

**Review by nesisknesis — APPROVED** [review]

**Comment by Cartlord** [comment]
I wish this could happen, but I doubt botany would be willing to ergularly co-operate with Medical, or that either department would actually contact the other to even try to make this happen.

**Comment by UselessTheremin** [comment]
> I wish this could happen, but I doubt botany would be willing to ergularly co-operate with Medical, or that either department would actually contact the other to even try to make this happen.

its also necessary to port wounds and a bunch of other cobbychem shit.

**Comment by SomeguyManperson** [comment]
don't care how little anyone thinks cooperation is impossible if it is that just means it's more  balanced

**Comment by ChesterTheCheesy** [comment]
Botany usually can't be arsed to grow anything for kitchen, so good luck with that.

**Comment by Cartlord** [comment]
Round 35125, I can't harvest a harvest-ready Aloe plant by clicking it with an empty hand.

**Comment by SomeguyManperson** [comment]
fixed that

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -134,18 +166,152 @@
...
 /obj/item/stack/medical/ointment/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is squeezing \the [src] into [user.p_their()] mouth! [user.p_do(TRUE)]n't [user.p_they()] know that stuff is toxic?</span>")
 	return TOXLOSS
+
+	/*
+	The idea is for these medical devices to work like a hybrid of the old brute packs and tend wounds,
+	they heal a little at a time, have reduced healing density and does not allow for rapid healing while in combat.
+	However they provice graunular control of where the healing is directed, this makes them better for curing work-related cuts and scrapes.
+
+	The interesting limb targeting mechanic is retained and i still believe they will be a viable choice, especially when healing others in the field.
```
Comment (Hopekz) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion
	The idea is for these medical devices to work as a hybrid of the old brute packs and tend wounds,
	they heal a little at a time, have reduced healing density, and do not allow for rapid healing while in combat.
	However they provide granular control of where the healing is directed, this makes them better for curing work-related cuts and scrapes.

	The interesting limb targeting mechanic is retained and I still believe they will be a viable choice, especially when healing others in the field.
```

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -134,18 +166,152 @@
...
+	repeating = TRUE
+	heal_brute = 3
+	heal_burn = 3
+	grind_results = list(/datum/reagent/consumable/aloejuice = 1)
+	merge_type = /obj/item/stack/medical/aloe
+
+/obj/item/stack/medical/poultice
+	name = "mourning poultices"
+	singular_name = "mourning poultice"
+	desc = "A type of primitive herbal poultice.\nWhile traditionally used to prepare corpses for the mourning feast, it can also treat scrapes and burns on the living, however, it is liable to cause shortness of breath when employed in this manner.\nIt is imbued with ancient wisdom."
```
Comment (Hopekz) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion
	desc = "A type of primitive herbal poultice.\nWhile traditionally used to prepare corpses for the mourning feast, it can also treat scrapes and burns on the living. However, it is liable to cause shortness of breath when employed in this manner.\nIt is imbued with ancient wisdom."
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -1943,8 +1943,16 @@
 	taste_description = "bananas"
 	can_synth = TRUE
 
+/datum/reagent/cellulose
+	name = "Cellulose Fibers"
+	description = "A crystaline polydextrose polymer, plants swear by this stuff."
```
Comment (Hopekz) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	description = "A crystalline polydextrose polymer, plants swear by this stuff."
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 4]
```diff
@@ -1943,8 +1943,16 @@
...
+	reagent_state = SOLID
+	color = "#E6E6DA"
+	taste_mult = 0
+
 /datum/reagent/lemoline
 	name = "Lemoline"
 	description = "Synthesized in off-station laboratories, used in several high quality medicines."
```
Comment (Hopekz) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	description = "Synthesized in off-station laboratories, used in several high-quality medicines."
```
